### DISCUSSION TITLE
Why do you support the integration of Ukraine into Russia?

### DISCUSSION THREAD

**Discussion by dukelec** [pnpm/pnpm#4431] [Q&A] [answered]
The chinese string "支持乌克兰加入俄罗斯" on the page:
https://pnpm.io/zh/ 
means: "Support the integration of Ukraine into Russia"

![Screenshot_20220311-165643_122](https://user-images.githubusercontent.com/4454365/157836531-ae7769f7-939c-4ebc-b131-2f3f7eda1eba.png)

**Comment by leohxj** [comment]
这谁干的, 是太闲了么

**Comment by BlackHole1** [comment]
<img width="891" alt="image" src="https://user-images.githubusercontent.com/8198408/157839754-47d7f27d-678d-4188-9fec-07692ced17aa.png">

User: https://crowdin.com/profile/Misaka-L / https://github.com/Misaka-L

I suggest disabling his access to the pnpm repo @zkochan

**Comment by lixeon** [comment]
lmao

**Reply by BlackHole1** [reply]
It's not funny 🙂

**Reply by xiaowj** [reply]
😄

**Comment by juanpicado** [comment, marked as answer]
I've blocked him and I've deployed the new website, the string will be replaced shortly

**Reply by xiaowj** [reply]
why？

**Reply by BlackHole1** [reply]
no why 🙂

**Reply by juanpicado** [reply]
Thanks for reporting, appreciated, I have no patience for :trollface: s.

**Reply by paulius-valiunas** [reply]
> why？

I guess because pnpm does not endorse terrorism

**Reply by zkochan** [reply]
Thank you all for your support and fast reaction!
